pkg/plugin/v1/init.go

```diff
@@ -81,12 +79,17 @@ func (p *initPlugin) BindFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&p.dep, "dep", true, "if specified, determines whether dep will be used.")
 	p.depFlag = fs.Lookup("dep")
 	fs.StringArrayVar(&p.depArgs, "depArgs", nil, "Additional arguments for dep")
-	fs.MarkDeprecated("dep", "use the fetch-deps flag instead")
-	fs.MarkDeprecated("depArgs", "will be removed with version 1 scaffolding")
+	if err := fs.MarkDeprecated("dep", "use the fetch-deps flag instead"); err != nil {
+		panic(err)
+	}
+	if err := fs.MarkDeprecated("depArgs", "will be removed with version 1 scaffolding"); err != nil {
+		panic(err)
+	}
 
 	// boilerplate args
 	fs.StringVar(&p.boilerplate.Path, "path", "", "path for boilerplate")
-	fs.StringVar(&p.boilerplate.License, "license", "apache2", "license to use to boilerplate.  May be one of apache2,none")
+	fs.StringVar(&p.boilerplate.License, "license", "apache2", "license to use to boilerplate. "+
+		"May be one of apache2,none")
```

Comment:
NIT: I would break between params:
```go
	fs.StringVar(&p.boilerplate.License, "license", "apache2",
		"license to use to boilerplate.  May be one of apache2,none")
```